node_modules/oae-core/editprofile/js/editprofile.js

```diff
@@ -111,9 +111,25 @@ define(['jquery', 'oae.core'], function ($, oae) {
             }
 
             // Render the form elements
+            // detect if is possible to modify the name and email
+            var disableModification = false;
+            var authenticationStrategy = oae.data.me.authenticationStrategy;
+           if ((authenticationStrategy === 'cas')| (authenticationStrategy === 'shibboleth')){
+            if (oae.api.config.getValue('oae-authentication', 'shibboleth', 'enabled')){
+                if (oae.api.config.getValue('oae-authentication', 'shibboleth', 'disableProfile')){
+                    var disableModification = true;                
+                }
+            }
+            if (oae.api.config.getValue('oae-authentication', 'cas', 'enabled')){
+                if (oae.api.config.getValue('oae-authentication', 'cas', 'disableProfile')){
+                    var disableModification = true;                   
```

Comment:
Same here: since the variable has already been declared we should take out `var`.